Main/Data/ApplicationDbContext.cs

```diff
@@ -180,6 +181,23 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
           .HasDefaultValueSql("(getdate())");
       });
 
+      modelBuilder.Entity<Configuration>(entity =>
+      {
+        entity.HasKey(e => new { e.Key })
+           .HasName("PK__Configuration__XOCTKZ6IAQ");
+
+        entity.ToTable("Configuration", "app");
+
+        entity.Property(e => e.Key)
+            .IsRequired()
+            .HasMaxLength(100)
+            .IsUnicode(false);
+
+        entity.Property(e => e.Value)
+            .IsRequired()
+            .HasMaxLength(100)
```

Comment:
I would give a much higher length for that. Never know what will be stored in the configuration in the future.